python/dgl/sparse/sparse_matrix.py

```diff
@@ -479,6 +479,71 @@ def is_diag(self):
         """Returns whether the sparse matrix is a diagonal matrix."""
         return self.c_sparse_matrix.is_diag()
 
+    def select(self, dim: int, ids: Union[tensor.Tensor, slice]):
```

Comment:
```suggestion
    def select(self, dim: int, rule: Union[tensor.Tensor, slice]):
```

Reply:
CI report error: "NameError: name 'Union' is not defined".